testing/jormungandr-automation/src/jormungandr/explorer/mod.rs

```diff
@@ -52,22 +53,49 @@ pub struct ExplorerProcess {
 }
 
 impl ExplorerProcess {
-    pub fn new(node_address: String, logs_dir: Option<std::path::PathBuf>) -> Self {
+    pub fn new(
+        node_address: String,
+        logs_dir: Option<std::path::PathBuf>,
+        params: ExplorerParams,
+    ) -> Self {
         let path = get_explorer_app();
         let explorer_port = get_available_port();
         let explorer_listen_address = format!("127.0.0.1:{}", explorer_port);
```

Comment:
this could be made a constant so that it's easier to change in the future

Reply:
Same as above, I'm planning to put all I can/makes sense in the ExplorerParams struct